base/sim/FairMCApplication.h

```diff
@@ -209,6 +215,16 @@ class FairMCApplication : public TVirtualMCApplication
      */
     FairMCApplicationState GetState() const { return fState; }
 
+    /**
+     * Add module to the list of sensitive detectors.
+     */
+    void AddSensitiveModule(std::string volName, FairModule* module);
+
+    /**
+     * Method introduced temporarily. It should go awway with DEPRACATED Bool_t FairDetector::ProcessHits()
+     */
+    FairVolume* GetFairVolume();
```

Comment:
What about using C++ deprecations?

```suggestion
    /**
     * \deprecate Method introduced temporarily. It should go awway with DEPRACATED Bool_t FairDetector::ProcessHits()
     */
    [[deprecated]] FairVolume* GetFairVolume();
```

(And doxygen deprecations in addition?)